docs/Accessibility.md

```diff
@@ -0,0 +1,75 @@
+---
+id: accessibility
+title: Accessibility
+layout: docs
+category: Guides
+permalink: docs/accessibility.html
+next: nativemodulesios
+---
+
+# Accessibility
+
+Accessibility on iOS encompasses many topics, but for many, accessibility is synonymous with VoiceOver, a technology available since iOS 3.0. It acts as a screen reader, allowing people with visual impairments to use their iOS devices. Click [here](https://developer.apple.com/accessibility/ios/) to learn more.
+
+## Making Accessible Apps
+
+### Coding Accessibly
+
+#### accessible
+
+When true, indicates that the view is an accessibility element. When a view is an accessibility element, it groups its children into a single selectable component. By default, all touchable elements are accessible.
```

Comment:
It basically means that they become a single selectable group. That is, you can't select a child of the group but you can select the group. For example, a table view cell may have multiple labels inside, but you would want the table view cell to be accessible so the cell itself becomes a single selectable element.